site/docs/components/slider.mdx

```diff
@@ -28,14 +28,21 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 
 <iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FeZKEE5kXbEMY3lx84oz8iN%2F%25E2%259D%25A4%25EF%25B8%258F-UI-Kit-Heart%3Fnode-id%3D16080%253A4049" allowfullscreen></iframe>
 
+## Options
+
+- **Default:**
+    - In a default slider, the user can make an imprecise selection from a range of values, where there is no implication of progression from one side to the other.
+- **Progress:**
+    - In a progress slider, the user can make an precise selection from a range of values, aided by an exact indication of value. A sense of progress is implied, from one side to the other.
+
 ## To keep in mind
 
 Sliders are helpful for choosing subjective values.
 
 - Always indicate the current, selected value. Ideally with visible text associated with the handle, but at a minimum provide a text alternative for screen readers.
 - Provide a visible label describing what the slider changes, such as "Brightness".
 - Show the minimum value on the left and maximum value on the right, where these are reversed in Right-To-Left (RTL) languages.
-- We generally use sliders with no fill. You might use a fill when it's useful to give the impression of building towards the maximum value.
+- Default sliders have no full. The progress slider uses a fill to give the impression of building towards the maximum value.
```

Comment:
@lindsey-collins should we be referencing the ###Progress track section at this point? e.g. See [Progress track](/components/slider/#progress-track)

Reply:
Good call! Updated.